src/actions/TouchActions.ts

```diff
@@ -1,10 +1,7 @@
-import {
-  getSupportedBiometryType,
-  loadFingerprintFile,
-  supportsTouchId
-} from '../keychain'
+import { getSupportedBiometryType, supportsTouchId } from '../keychain'
 import { TouchState } from '../reducers/TouchReducer'
 import { Dispatch } from '../types/ReduxTypes'
+import { readKeychainFile } from '../util/keychainFile'
```

Comment:
Where is this? This doesn't seem to resolve for me

Reply:
This never got resolved. I have a broken build doing `yarn prepare` in the repo.